internal/sql/repository/pipelineConfig/CiWorkflowRepository.go

```diff
@@ -87,8 +87,24 @@ type GitCommit struct {
 	Date    time.Time
 	Message string
 	Changes []string
+	PrData PrData
 }
 
+type PrData struct {
```

Comment:
This is not a  table,shouldnt exist here also its duplicate of struct in app.go

Reply:
This is problem I can see in the current code. We are not defining bean in a common package. 
I have copied the same struct because of the circular dependencies of the packages. 
I tried to solve it on the first place but couldn't solve it. so decided to copy the struct .

(Maybe due the same problem - GitCommit struct is copied everywhere.) This is the issue.